doc/htmldoc/installation/cmake_options.rst

```diff
@@ -87,6 +87,11 @@ External libraries
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
 | ``-Dwith-gsl=[OFF|ON|</path/to/gsl>]``                | Build with the GSL library [default=ON]. To set a specific library, give the install path.     |
 +-------------------------------------------------------+------------------------------------------------------------------------------------------------+
++-------------------------------------------------------+------------------------------------------------------------------------------------------------+
+| ``-Dwith-hdf5=[OFF|ON|</path/to/hdf5>]``              | Build with `HDF5 <https://hdfgroup.org/>`_ library [default=OFF]. To set a specific library,   |
+|                                                       | give the install path. HDF5 is needed to configure NEST for                                    |
+|                                                       | `SONATA <https://github.com/AllenInstitute/sonata>`_ support.                                  |
```

Comment:
```suggestion
| ``-Dwith-hdf5=[OFF|ON|</path/to/hdf5>]``              | Build with `HDF5 <https://hdfgroup.org/>`_ library [default=OFF]. To set a specific library,   |
|                                                       | give the install path. HDF5 is required for                                    |
|                                                       | `SONATA <https://github.com/AllenInstitute/sonata>`_ support.                                  |
```

I think it would be better to link to `nest_sonata_guide.rst` here instead of to the generic sonata homepage. That will lead the user to more relevant documentation.